R/nix_hash.R

```diff
@@ -92,7 +92,7 @@ hash_url <- function(url) {
   sri_hash <- nix_sri_hash(path = path_to_source_root)
 
   paths <- list.files(path_to_src, full.names = TRUE, recursive = TRUE)
```

Comment:
I wonder if it wouldn’t be enough to remove `recursive = TRUE` instead could you try this?

Reply:
Not that s not enough because the path is hidden in the tmp folder, so just removing recursive leads to no match.